database/mssql/scripts/versions/v_41_ddl.sql

```diff
@@ -0,0 +1,98 @@
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+GO
+
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0
+   SET NOEXEC ON
+GO
+
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD [PREVIOUS_LOA_ID] [int] NULL
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD  [ORIGINAL_LOA_ID] [int] NULL
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] ALTER COLUMN [START_DATE] [date] NOT NULL
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] ALTER COLUMN [EXPIRY_DATE] [date] NULL
+ ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] DROP CONSTRAINT ORBC_LOA_PERMIT_TYPES_PERMIT_TYPE_ID_FK
+ EXEC sp_rename @objname = 'permit.ORBC_LOA_PERMIT_TYPE_DETAILS.PERMIT_TYPE_ID', @newname = 'PERMIT_TYPE', @objtype = 'COLUMN'
+ ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS]
+   WITH CHECK ADD CONSTRAINT [FK_ORBC_LOA_PERMIT_TYPES_PERMIT_TYPE] FOREIGN KEY ([PERMIT_TYPE]) REFERENCES [permit].[ORBC_PERMIT_TYPE]([PERMIT_TYPE])
+ ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] CHECK CONSTRAINT [FK_ORBC_LOA_PERMIT_TYPES_PERMIT_TYPE]
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] WITH CHECK ADD CONSTRAINT [FK_ORBC_LOA_DETAILS_PREVIOUS_LOA_ID] FOREIGN KEY([PREVIOUS_LOA_ID]) 
+ REFERENCES [permit].[ORBC_LOA_DETAILS] ([LOA_ID])
+ GO
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] CHECK CONSTRAINT [FK_ORBC_LOA_DETAILS_PREVIOUS_LOA_ID]
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] WITH CHECK ADD CONSTRAINT [FK_ORBC_LOA_DETAILS_ORIGINAL_LOA_ID] FOREIGN KEY([ORIGINAL_LOA_ID]) 
+ REFERENCES [permit].[ORBC_LOA_DETAILS] ([LOA_ID])
+ GO
+ ALTER TABLE [permit].[ORBC_LOA_DETAILS] CHECK CONSTRAINT [FK_ORBC_LOA_DETAILS_ORIGINAL_LOA_ID]
+ ALTER TABLE [permit].[ORBC_LOA_VEHICLES] DROP CONSTRAINT [ORBC_LOA_VEHICLES_LOA_ID_FK]
+ ALTER TABLE [permit].[ORBC_LOA_VEHICLES]
+   WITH CHECK ADD CONSTRAINT [ORBC_LOA_VEHICLES_LOA_ID_FK] FOREIGN KEY ([LOA_ID]) REFERENCES [permit].[ORBC_LOA_DETAILS]([LOA_ID]) 
+ALTER TABLE [permit].[ORBC_LOA_VEHICLES] CHECK CONSTRAINT [ORBC_LOA_VEHICLES_LOA_ID_FK]
+ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] DROP CONSTRAINT [ORBC_LOA_PERMIT_TYPE_LOA_ID_FK]
+ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS]
+   WITH CHECK ADD CONSTRAINT [ORBC_LOA_PERMIT_TYPE_LOA_ID_FK] FOREIGN KEY ([LOA_ID]) REFERENCES [permit].[ORBC_LOA_DETAILS]([LOA_ID]) 
```

Comment:
Should be FK_ORBC_LOA_PERMIT_TYPE_LOA_ID as per Moti std. Please rectify other FK from initial LOA ddl script as well if any